extensions/blocks/contact-form/editor.scss

```diff
@@ -20,14 +20,18 @@
 	}
 
 	.help-message {
+		color: $dark-gray-150;
 		width: 100%;
-		margin: -18px 0 28px;
+		margin: 0 0 1em;
 	}
 
 	.components-base-control {
-		margin-bottom: 16px;
 		width: 100%;
 	}
+
+	.components-button {
+		margin: 6px 0 0;
```

Comment:
This seems like a case of us overriding the default styles without good reason.  I think we can just remove it.

Reply:
Good call. Removed in https://github.com/Automattic/jetpack/commit/fe080710f57fdade89a28cab5dedf38525804336